taxonomies/categories.txt

```diff
@@ -22564,17 +22566,14 @@ fr:Crème Fraîche d'Isigny
 
 <en:Creams
 <en:Fermented milk products
-en:Sour creams, Crèmes fraîches
-de:Sauerrahm
+en:Sour creams, Crèmes fraîches, Smetana
+de:Sauerrahm, Schmand, Saure Sahne
 fr:Crèmes fraîches, Crème fraîche
 hu:Tejföl
 nl:Zure romen
+ru:Сметана
 pnns_group_2:en:Milk and yogurt
 
```

Comment:
I do not like to put everything together. In some countries they are really different, either by habit or regulation. They could all be a child of "en:fermented creams", so that we can view and analyse them as one group.